engines/scumm/saveload.cpp

```diff
@@ -1137,6 +1137,11 @@ void ScummEngine::saveOrLoad(Serializer *s) {
 	if (s->isLoading() && s->getVersion() < VER(14))
 		upgradeGfxUsageBits();
 
+	// When loading, reset the setShakePos. Fixes bug #7141 Partially
```

Comment:
And as a meta-comment, references to bugs generally fit better in commit messages than in comments. (Since the code comment should describe the code as it is, not compared to some older broken state.)